code/modules/antagonists/abductor/equipment/gland.dm

```diff
@@ -478,8 +478,8 @@
 /obj/item/organ/heart/gland/plasma/proc/vomit_plasma()
```

Comment:
Please rename all instances of plasma pertaining to this organ to either miasma or something more vague so the type of gas doesn't matter

Reply:
Alright. I didn't do that originally because I was worried about modular support for the upcoming changes.

Reply:
> Alright. I didn't do that originally because I was worried about modular support for the upcoming changes.

When this is done we can merge it.

Reply:
done